crates/virtio-queue/CHANGELOG.md

```diff
@@ -1,10 +1,19 @@
-# Unreleased
+# v0.3.0
 
 ## Added
-- QueueStateT::size(): [#153](https://github.com/rust-vmm/vm-virtio/pull/153)
+- `QueueStateOwnedT` trait that stands for queue objects which are exclusively
+  owned and accessed by a single thread of execution [#148](https://github.com/rust-vmm/vm-virtio/pull/148)  
+- `QueueStateT::pop_descriptor_chain(), QueueGuard::pop_descriptor_chain()`,
+  `QueueGuard::go_to_previous_position()`: [#148](https://github.com/rust-vmm/vm-virtio/pull/148)
+- `MockSplitQueue::add_desc_chain()`: [#151](https://github.com/rust-vmm/vm-virtio/pull/151)
+- `QueueStateT::size()`: [#153](https://github.com/rust-vmm/vm-virtio/pull/153)
+
+## Changed
+- The minimum version of the `vm-memory` dependency is now `v0.8.0`.
+- Improve the efficiency of `needs_notification` [#161](https://github.com/rust-vmm/vm-virtio/pull/161)
 
 ## Removed
-- #[derive(Clone)] for QueueState: [#153](https://github.com/rust-vmm/vm-virtio/pull/153)
+- `#[derive(Clone)]` for `QueueState`: [#153](https://github.com/rust-vmm/vm-virtio/pull/153)
```

Comment:
This is the first release where we specify the links to the PRs/issues. Maybe we can keep the same format as the other crates: https://github.com/rust-vmm/vm-memory/blob/main/CHANGELOG.md

Reply:
Done!